packages/playwright-test/src/matchers/toMatchSnapshot.ts

```diff
@@ -212,19 +212,20 @@ class SnapshotHelper<T extends ImageComparatorOptions> {
     else
       output.push('');
 
+    const targetPath = path.relative(process.cwd(), this.snapshotPath);
```

Comment:
`process.cwd()` should never be used in matchers (and generally, we prefer to never use is at all).